tools/nemo_forced_aligner/utils/constants.py

```diff
@@ -16,4 +16,4 @@
 
 SPACE_TOKEN = "<space>"
 
-V_NEGATIVE_NUM = -1e30
+V_NEGATIVE_NUM = -3.4e38  # this is just above the most negative number in torch.float32
```

Comment:
Why use such a small value ? For all intents and purposes, -1e30 is small enough

Reply:
Yes, I think you are right that -1e30 is small enough. I think my reasoning was that -1e30 is arbitrary and -3.4e38 is less arbitrary since it is close to the limit. Is there any downside to making it this small?